localization/react-intl/src/app/components/dashboard/VerticalBarFactChecksByRating.json

```diff
@@ -3,5 +3,10 @@
     "id": "verticalBarFactChecksByRating.title",
     "description": "Title for the number of fact-checks by rating widget",
     "defaultMessage": "Claim & Fact-Checks"
+  },
+  {
+    "id": "verticalBarFactChecksByRating.removed",
+    "description": "Label for the fact-checks ratings chart for a rating that was removed",
+    "defaultMessage": "(removed)"
```

Comment:
where's this string used? was it included in another PR?

Reply:
Oh, it shouldn't be here anymore... it was me testing an alternative... let me clean it up.